src/pages/Login/index.js

```diff
@@ -57,21 +57,30 @@ function Login() {
 
     return (
         <>
-            <div className="Login">
-                <div>
-                    <div className="loginlogo">
-                        <img src={logo} />
-                    </div>
-                    <div
-                        id="customBtn"
-                        className="customGPlusSignIn"
-                        onClick={handleClick}
-                    >
-                        <span className="icon"></span>
-                        <span className="buttonText">Sign in with Google</span>
+            {accessToken ? (
+                <Redirect to="/" />
+            ) : (
+                <div className="Login">
+                    <div>
+                        <div className="loginlogo">
+                            <img src={logo} alt="code library logo" />
+                        </div>
+                        <div
+                            id="customBtn"
+                            className="customGPlusSignIn"
+                            onClick={handleClick}
+                            onKeyDown={handleClick}
+                            role="button"
+                            tabIndex={0}
+                        >
+                            <span className="icon" />
+                            <span className="buttonText">
+                                Sign in with Google
+                            </span>
+                        </div>
```

Comment:
1. We should probably extract the Login component to a presentational one and import it here
2. Why aren't we using the chakra ui when creating the component but instead html tags ?